e2e/utils/DebugDriver.ts

```diff
@@ -22,6 +22,32 @@ export const setupFork = async (testInfo: TestInfo) => {
   process.env[`TENDERLY_FORK_ID_TEST_${testInfo.testId}`] = fork.id;
 };
 
+export const setupLocalStorage = async (page: Page, testInfo: TestInfo) => {
+  const forkId = process.env[`TENDERLY_FORK_ID_TEST_${testInfo.testId}`];
+  if (!forkId)
+    throw new Error('Fork should be created before setting theRC URL');
+  const rpcUrl = forkRpcUrl(forkId);
+  // We need to be on a page to set localstorage so we create an empty page
+  await page.route(testInfo.testId, (route) => {
+    return route.fulfill({ status: 200, contentType: 'text/plain', body: '' });
+  });
+  await page.goto(testInfo.testId);
+
+  return page.evaluate((rpc) => {
+    // each value is stringified to match lsservice
+    localStorage.setItem('carbon-v1.1-tenderlyRpc', `"${rpc}"`);
+    localStorage.setItem('carbon-v1.1-isUncheckedSigner', 'true');
+    localStorage.setItem(
+      'carbon-v1.1-carbonControllerAddress',
+      '"0xC537e898CD774e2dCBa3B14Ea6f34C93d5eA45e1"'
+    );
+    localStorage.setItem(
+      'carbon-v1.1-voucherContractAddress',
+      '"0x3660F04B79751e31128f6378eAC70807e38f554E"'
+    );
+  }, rpcUrl);
```

Comment:
I would prefer if these values come from a separate initialLocalStorage file to make it easier to update with more initial localStorage data